### PR TITLE
V1.2.1.0

### PR DESCRIPTION
**Pull request by akzhy**
## V1.2.1.0

SEO Optimized for existing pages.
No visual changes

### CODE REVIEW

**Comment by Athul-CA** [comment]
I don't see the plugin 🙈🙈 and we could have updated the header rules using the `manifest` plugin

**Comment by Athul-CA** [comment]
Sorry `manifest` is for making this a pwa. 😅😅Good Job

File: src/components/indexWelcome.js [hunk 1]
```diff
@@ -1,6 +1,12 @@
 import React from "react"
 import { StaticQuery, graphql } from "gatsby"
 import marked from "marked"
+import {Helmet} from "react-helmet"
+import { globalHistory } from "@reach/router"
```
Comment (Athul-CA) [inline src/components/indexWelcome.js]: Is it `reach` or `react`?

**Comment by Athul-CA** [comment]
And lemme try a 404 page

**Comment by akzhy** [comment]
404 pages have not been added yet.

File: src/components/indexWelcome.js (hunk 1 above)
Comment (akzhy) [reply src/components/indexWelcome.js]: Its `reach`

**Comment by Athul-CA** [comment]
I made one. Lemme PR It... Do check on the PR will you

**Pull request merged**